Cart_CleanCodePractices.Tests/CartTest.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Cart_CleanCodePractices.Tests
+{
+    public class CartTest
+    {
+        [Fact]
+        public void AddProductToCartTest()
+        {
+            Product cornFlakes = new Product("Corn Flakes",125);
+            Product milk = new Product("Milk", 48);
+            Cart cart = new Cart();
+            Dictionary<Product, int> expectedList = new Dictionary<Product, int>();
+
+            cart.Add(cornFlakes, 2);
+            expectedList.Add(cornFlakes, 2);
+
+            cart.Add(milk, 5);
+            expectedList.Add(milk, 5);
+
+            var actualList = cart.GetItemList();
+
+            Assert.Equal(expectedList,actualList);
```

Comment:
After discussing with Mr. Kumar, this test case will always fail as expectedList and actualList reffers to different objects.

Reply:
Same for all other test cases

Reply:
This test case was passing every time!

Reply:
I think we can discuss this tomorrow. I know that both the variables are pointing to two different objects, but in my case, as the objects are having same signature and equal values, the comparison results as True.

Reply:
@tavisca-nrajpurohit  This will work as xunit supports collection asserts.But it will not work if you change order of elements.